GruntFile.js

```diff
@@ -132,7 +132,8 @@ module.exports = function(grunt) {
                         'bower_components/d3/d3.js',
                         'src/*.js',
                         'src/models/*.js',
-                        'test/mocha/*.coffee'
+                        'test/mocha/*.coffee',
+	                    'https://cdn.rawgit.com/Kcnarf/d3-beeswarm/fbda9b54/build/d3-beeswarm.min.js',
```

Comment:
It's just for the tests right?  If so then it's probably fine, we also pull d3 externally, I was mostly curious what it was for.  Probably a good idea to add a comment there in the grunt file about what it is though just to keep track of it.